bemani/backend/iidx/pendual.py

```diff
@@ -889,46 +900,40 @@ def handle_IIDX22pc_request(self, request: Node) -> Optional[Node]:
 
             limit = Node.void('limit')
             root.add_child(limit)
-            limit.set_attribute('phase', '24')
+            limit.set_attribute('phase', '21')
 
             # See if we configured event overrides
             if self.machine_joined_arcade():
                 game_config = self.get_game_config()
                 timeshift_override = game_config.get_int('cycle_config')
-                omni_events = game_config.get_bool('omnimix_events_enabled')
+                ccxbm_on = game_config.get_bool('ccxbm_enable')
             else:
                 # If we aren't in an arcade, we turn off events
                 timeshift_override = 0
-                omni_events = False
-
-            if self.omnimix and (not omni_events):
-                boss_phase = 0
-            else:
-                # TODO: Figure out what these map to
-                boss_phase = 0
+                ccxbm_on = False
 
+            #events
             boss = Node.void('boss')
             root.add_child(boss)
-            boss.set_attribute('phase', str(boss_phase))
+            boss.set_attribute('phase', str(game_config.get_int('event_phase')))
 
             chrono_diver = Node.void('chrono_diver')
             root.add_child(chrono_diver)
             chrono_diver.set_attribute('phase', '3')
 
             qpronicle_chord = Node.void('qpronicle_chord')
             root.add_child(qpronicle_chord)
-            qpronicle_chord.set_attribute('phase', '2')
+            qpronicle_chord.set_attribute('phase', '3')
 
             common_cd_event = Node.void('common_cd_event')
             root.add_child(common_cd_event)
-            common_cd_event.set_attribute('open_list', '3')
+            common_cd_event.set_attribute('open_list', str(3))
```

Comment:
Why this change?

Reply:
I had messed around with changing the values with frontend, meant to set it back to just being a string.